.github/workflows/ci.yml

```diff
@@ -81,6 +81,15 @@ jobs:
       - name: "Mypy check"
         run: pipx run --python 3.12 mypy
 
+  lint-other:
+    name: "Lint other"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: "Check documentation formatting"
+        run: npx prettier --prose-wrap always --check docs/*.md docs/concepts/**/*.md docs/guides/**/*.md docs/pip/**/*.md
```

Comment:
Why not `docs/**/*.md`? Because we need to exclude generated files? This seems brittle.

Reply:
Yeah, i went with the conservative option here. I have touched neither the reference docs nor the changelog, so i'll follow your lead for them.

Reply:
Does it support exclusions? Can we encode this in a repo-level config file?

Reply:
I added exclusion comments. If we have more setting prettier we can switch to one of their configuration files.

I'd like to format the generated files too, but the diff is already large enough.